gluetool_modules/dep_list.py

```diff
@@ -97,6 +97,7 @@ def pip_version_unify(self, pip_deps):
     def get_allowed_version_bounds(self, item):
         if not item.equal and not item.lower and not item.upper:
             return item.pkg
+        # pylint: disable=no-else-return
```

Comment:
This is actually something we should rather fix than silence.

```python
        if item.equal:
            ...
            return '{}=={}'.format(item.pkg, item.equal)
        elif item.upper and item.lower:
```

would become:

```python
        if item.equal:
            ...
            return '{}=={}'.format(item.pkg, item.equal)
        if item.upper and item.lower:
```

`elif` is not necessary, the previous condition ends with `return`, therefore either the condition was true-ish, then you leave the function through this condition, or it was false-ish, and you can test again with plain `if`.

Reply:
right, was not getting it yesterday, thanks

Reply:
fixed